p-token/Cargo.toml

```diff
@@ -1,14 +1,12 @@
 [package]
-name = "token-program"
+name = "pinocchio-token-program"
```

Comment:
I see you already snagged the spl-token-interface crate name, but be sure to reserve this one too!